fmc/src/flow/crypto.rs

```diff
@@ -0,0 +1,23 @@
+/*++
+
+Licensed under the Apache-2.0 license.
+
+File Name:
+
+    crypto.rs
+
+Abstract:
+
+    Crypto helper routines
+
+--*/
+use caliptra_drivers::{Ecc384PubKey, KeyId};
+/// DICE  Layer Key Pair
+#[derive(Debug)]
+pub struct Ecc384KeyPair {
```

Comment:
Should probably move this into the driver crate and then just use it in both ROM and FMC

Reply:
I agree, but  Vishal is  trying to minimize code sharing and driver crate is supposed to be ROM only. I can move the type to the common crate.

Reply:
Done